osdk/src/util.rs

```diff
@@ -163,15 +163,19 @@ pub fn get_current_crate_info() -> CrateInfo {
     let metadata = get_cargo_metadata(None::<&str>, None::<&[&str]>).unwrap();
 
     let default_member = get_default_member(&metadata);
+    parse_package_id_string(default_member)
+}
+
+pub fn parse_package_id_string(package_id: &str) -> CrateInfo {
     // Prior 202403 (Rust 1.77.1), the default member string here is in the form of
```

Comment:
```suggestion
    // Prior 202403 (Rust 1.77.1), the package id string here is in the form of
```

Comments are needed to be synced here.